src/main/java/gameframework/gui/GameWindow.java

```diff
@@ -17,17 +17,18 @@
 	protected final ObservableValue<Integer> score;
 	protected final ObservableValue<Integer> life;
 
-	public GameWindow(GameCanvas gameCanvas, GameConfiguration configuration,
-			ObservableValue<Integer> score, ObservableValue<Integer> life) {
+	public GameWindow(String gameName, GameCanvas gameCanvas,
```

Comment:
could you please keep the old constructor for backward compatibility? You can add the `@Deprecated` annotation on the old constructor.